docs/data-sources/gateway_control_plane_list.md

```diff
@@ -0,0 +1,147 @@
+---
```

Comment:
I've included the data source documentation for control_plane_list - please let me know if not needed